src/vs/workbench/contrib/customEditor/browser/customEditor.contribution.ts

```diff
@@ -34,19 +33,5 @@ Registry.as<IEditorInputFactoryRegistry>(EditorExtensions.EditorInputFactories)
 		CustomEditorInputSerializer.ID,
 		CustomEditorInputSerializer);
 
-Registry.as<IEditorInputFactoryRegistry>(EditorExtensions.EditorInputFactories)
-	.registerCustomEditorInputFactory(Schemas.vscodeCustomEditor, customEditorInputFactory);
-
-
-/**
- * Register a workbench contribution so that the custom editor service gets registered on startup.
- * This is needed for CLI opening of custom editors
- */
-class WorkbenchConfigurationContribution {
-	constructor(
-		@ICustomEditorService _customEditorService: ICustomEditorService,
```

Comment:
@lramos15 fyi I moved this "hack" into the new `ComplexCustomWorkingCopyEditorHandler`